nilearn/experimental/surface/_maskers.py

```diff
@@ -226,10 +220,10 @@ def fit(
 
         Parameters
         ----------
-        img : SurfaceImage object
+        img
             Mesh and data for both hemispheres.
 
-        y : None
+        y
             This parameter is unused. It is solely included for scikit-learn
             compatibility.
 
```

Comment:
rendering of SurfaceLabelsMasker.fit ( we would render the input parameter types in the signature for this option but I somehow messed up the configs here so use your imagination :) )
![image](https://github.com/nilearn/nilearn/assets/63292494/c1b4a40a-6215-4d35-ab54-6e7075d661fa)